lib/uglifier.js

```diff
@@ -105,16 +130,20 @@ function processAssets(assetHash, options) {
   });
 
   return Promise.all(promises).then(() => {
+    // build is done, clean up the cache
+    pruneCache(usedCacheKeys, getCacheKeysFromDisk(options.cacheDir), options.cacheDir);
```

Comment:
I don't think I have a clear mental model of all that is going on here. Is the cache pruned on all builds or just master builds? Should it only be master builds?